ecleankernel/file.py

```diff
@@ -122,6 +125,9 @@ def decompress_raw(self, f: typing.IO[bytes]) -> bytes:
             b'\x89\x4c\x5a\x4f\x00\x0d\x0a\x1a\x0a': 'lzo',
         }
         maxlen = max(len(x) for x in magic_dict)
+        if size and maxlen > size:
+            return f.read(size)
+        size = size or -1
```

Comment:
I had to keep this to make mypy happy. It complained:

```
error: Argument 1 to "read" of "IO" has incompatible type "int | None"; expected "int"  [arg-type]
```

Reply:
Hmm, docs say it can be either omitted or negative, so this is more correct.

That said, since we no longer support any streaming variant now, I suppose you could move `f.read(size)` earlier and make it shared between all variants.

Reply:
Done.